app.rb

```diff
@@ -1,8 +1,30 @@
 require 'sinatra/base'
+require './lib/peep'
+require './database_connection_setup'
 
 class Chitter < Sinatra::Base
-  get '/test' do
-    'Test page'
+
+  enable :sessions
+
+  get '/peeps' do
```

Comment:
Nice work integrating the search functionality into this route!

Looks a bit like you're going to filter the peeps in the view.  You could also do it here and then pass @peeps, filtered or not, into the view.

Reply:
that would mean you only need one view template

Reply:
you could also create a new method on the peep class to filter the results at the DB level